app/controllers/idv/personal_key_controller.rb

```diff
@@ -82,5 +84,11 @@ def in_person_enrollment?
     def pending_profile?
       current_user.pending_profile?
     end
+
+    def device_profiling_failed?
+      return false unless IdentityConfig.store.proofing_device_profiling_collecting_enabled
```

Comment:
This should be `proofing_device_profiling_decisioning_enabled`, yes?
```suggestion
      return false unless IdentityConfig.store.proofing_device_profiling_decisioning_enabled
```